### PR TITLE
Jcv v0.3.0

### PR DESCRIPTION
**Pull request by jcvernaleo**

### CODE REVIEW

**Comment by marcopeereboom** [comment]
ok

**Comment by alexlyp** [comment]
OK

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+| decred/decred-windows-installer | 5d0fad99eee1eb3afdd2ba9f1fe54f8a29fb8547 |
+| gominer | 093470d6b738a290dff1e32905b1b82b6fd99b32 |
+
+## Known Issues
+
+### Paymetheus
+
+* Spendable balance is incorrect and may even go negative after purchasing tickets. This is only a temporary accounting issue and will resolve itself after the tickets have been mined into a block.
+
+dcrwallet
```
Comment (jrick) [inline release-notes.md]: Needs the markdown header

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,110 @@
...
+
+* The [spentnessnotifiations](https://github.com/decred/dcrwallet/blob/master/rpc/documentation/api.md#spentnessnotifications) gRPC API has been intentionally broken to avoid a deadlock situation. The RPC will appear to be working  but no responses will be streamed to the client.
```
Comment (jrick) [inline release-notes.md]: sorry, I edited my original comment.  The RPC name should technically be WalletService.SpentnessNotifications (and this time I spelled notifications correctly).

**Comment by jrick** [comment]
ok

**Pull request merged**